### PR TITLE
yacht task

### PR DESCRIPTION
**Pull request by axtens**
Again, not on @KTSnowy 's list but was doing it for 8th.

### CODE REVIEW

File: exercises/practice/yacht/test.ps1 [hunk 1]
```diff
@@ -0,0 +1,14 @@
+if (![System.IO.File]::Exists("$PSScriptRoot/bin/cobolcheck.exe")){
+  Write-Output "Cobolcheck not found. Trying to fetch it."
+  & "$PSScriptRoot/bin/fetch-cobolcheck.ps1"
+}
+
+Write-Output "Run cobolcheck."
+Set-Location $PSScriptRoot
+
+Invoke-Expression "bin/cobolcheck.exe -p hello-world"
```
Comment (0xE282B0) [inline exercises/practice/yacht/test.ps1:9]:
```suggestion
Invoke-Expression "bin/cobolcheck.exe -p yacht"
```

File: exercises/practice/yacht/test.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )/
+
+if [ ! -f $SCRIPT_DIR/bin/cobolcheck ]; then
+    echo "Cobolcheck not found, try to fetch it."
+    cd $SCRIPT_DIR/bin/
+    ./fetch-cobolcheck
+fi
+cd $SCRIPT_DIR
+./bin/cobolcheck -p hello-world
```
Comment (0xE282B0) [inline exercises/practice/yacht/test.sh:10]:
```suggestion
./bin/cobolcheck -p yacht
```

**Review by 0xE282B0 — COMMENTED** [review]
[exercises/practice/yacht/bin/fetch-cobolcheck](https://github.com/exercism/cobol/pull/25/files#diff-81327fc5f652058441e6eec6eee7665bca3cd901a02193d82793aba6cc75b502) needs to be executable as well.

Do we want to provide `WS-DICE`, `WS-CATEGORY` and `WS-RESULT` in the stub file?

**Comment by axtens** [comment]
Yes including those in the stub would be good though reading the tests is encouraged across the site

**Pull request closed without merging**